bindings/python_cffi/zyre_cffi/_cdefs.inc

```diff
@@ -93,14 +93,6 @@ void
 int
     zyre_set_endpoint (zyre_t *self, const char *format, ...);
 
-//
```

Comment:
ah i know what i did.. when you asked me to re-add that stuff i had already re-gen'd it in my curve branch... so instead of clearing it out first, ... i just mashed buttons. #fail